src/app/shared/components/button/button.component.html

```diff
@@ -0,0 +1,13 @@
+<ng-container [formGroup]="group">
+  <div class="form-group">
+    <button
+      class="btn btn-{{ field.colorClass }}"
+      [class.field.colorClass]="true"
+      type="submit"
+      [disabled]="isDisabled"
```

Comment:
[textContent]='field.label'

Reply:
in this case dont agree, because we will have different structure from other forms component, and also less obvious